setup.py

```diff
@@ -47,11 +35,11 @@
 
     # versioneer is necessary in order to update (but *not* merely to
     # use) the automatic versioning tools.
-    "versioneer==0.10",
+    "versioneer==0.15",
 
     # Some of the tests use Conch:
     "PyCrypto==2.6.1",
-    "pyasn1==0.1.7",
+    "pyasn1",
```

Comment:
Why does this no longer have a pinned version? Either pin a version or add a comment.